{"@class": "de.fau.cs.inf2.cthree.data.Cluster", "id": "5bcc50bfe99da2037655a97bce1931bd", "detectedBy": ["AST_DBSCAN"], "members": [{"@class": "de.fau.cs.inf2.cthree.data.CodeChange", "id": "0bf1d74b9af03de7897d3837006ab07b", "repository": "https://github.com/junit-team/junit.git", "fileName": "src/test/java/org/junit/runner/JUnitCommandLineParserTest.java", "commitBeforeChange": "cfaceb6d34390d999ff238d53fcb73898b32f98f", "commitAfterChange": "fa5df4e972db64249046b789b3dc730657f11c58", "methodNumberBeforeChange": 5, "methodNumberAfterChange": 5, "signatureBeforeChange": "\r \r     @Test\r     public void shouldCreateFailureUponUnknownOption() throws Exception", "signatureAfterChange": "\r \r     @Test\r     public void shouldCreateFailureUponUnknownOption() throws Exception", "diff": ["-        assertThat(description.toString(), containsString(\"initializationError:\"));\r", "+        assertThat(description.toString(), containsString(\"initializationError\"));\r"]}, {"@class": "de.fau.cs.inf2.cthree.data.CodeChange", "id": "6b5a062ff8709b42fb6c39c26076cb7e", "repository": "https://github.com/junit-team/junit.git", "fileName": "src/test/java/org/junit/tests/experimental/interceptor/ExpectedExceptionInterceptorTest.java", "commitBeforeChange": "3f05aac1da5851a1957c3c9ca46148464e5fd513", "commitAfterChange": "a423117429dc71bf3068eeae38c6443672beda7f", "methodNumberBeforeChange": 7, "methodNumberAfterChange": 7, "signatureBeforeChange": "  \t@Test \tpublic void wrongMessageFails()", "signatureAfterChange": "  \t@Test \tpublic void wrongMessageFails()", "diff": ["-\t\t\t\thasSingleFailureContaining(\"Unexpected exception message, expected<expectedMessage> but was<actualMessage>\"));", "+\t\t\t\thasSingleFailureContaining(\"\\\"expectedMessage\\\"\\n     but: getMessage() was \\\"actualMessage\\\"\"));"]}]}